llvm/include/llvm/Transforms/Vectorize/SandboxVectorizer/Passes/BottomUpVec.h

```diff
@@ -25,7 +25,9 @@ namespace llvm::sandboxir {
 class BottomUpVec final : public FunctionPass {
   bool Change = false;
   std::unique_ptr<LegalityAnalysis> Legality;
-  void vectorizeRec(ArrayRef<Value *> Bndl);
+
+  Value *createVectorInstr(ArrayRef<Value *> Bndl, ArrayRef<Value *> Operands);
```

Comment:
A one-line comment explaining what the arguments mean would be nice.

Reply:
Done